demos/browser/package-lock.json

```diff
@@ -1,18 +1,25 @@
 {
   "name": "amazon-chime-sdk-js-demo",
   "version": "2.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
Guess this should be `1`. I could update this if needed.

Reply:
Nope 2 is correct actually, I this npm > 8 will use 2 as the lockfileVersion.